code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -106,8 +106,8 @@
 	name = "light tiles"
 	gender = PLURAL
 	singular_name = "light floor tile"
-	desc = "A floor tile, made out off glass. Use a multitool on it to change its color."
-	icon_state = "tile_light blue"
+	desc = "A floor tile made of glass. Use a multitool on it to change its color."
```

Comment:
```suggestion
	desc = "A floor tile made of glass, with an integrated light. Use a multitool on it to change its color."
```

Nothing major, but it _is_ more than just a glass tile. The rest of the grammar looks fine.